docs/source/faq.rst

```diff
@@ -202,6 +202,9 @@ To make the parameters suggested by Optuna reproducible, you can specify a fixed
     study = optuna.create_study(sampler=sampler)
     study.optimize(objective)
 
+To make the pruning by :class:`~optuna.pruners.HyperbandPruner` reproducible, you can specify ``study_name`` of :class:`~optuna.study.Study`.
```

Comment:
[NITS]
```suggestion
To make the pruning by :class:`~optuna.pruners.HyperbandPruner` reproducible, please specify a fixed ``study_name`` of :class:`~optuna.study.Study` in addition to the ``seed`` argument.
```
The phrase "you can" may imply that specifying the option is optional. However, in this case, reproducibility is not guaranteed unless both the seed and the study_name are set. Therefore, I prefer a stronger expression.

Reply:
Thanks!